blog/2025-07-12.md

```diff
@@ -0,0 +1,68 @@
+---
+title: '📢 Ed-Fi Data Management Service Release Candidate Now Available!'
+authors: mayya
+tags: [tanager, release]
+---
+
+We’re thrilled to announce the **Release Candidate 1 / v0.6.0 release**
+milestone for the [Ed-Fi Data Management Service
+(DMS)](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/). This
+release introduces Swagger UI, template databases, and completing critical
+end-to-end and compatibility testing. It also lays the groundwork for smoother
+integration and a better developer experience.
+
+<!-- truncate -->
+
+## 📌 Key Updates
+
+1. **Swagger UI**: Adds interactive API documentation support to explore and
+   test DMS endpoints with ease.
+2. **Minimal and Populated Templates**: Provides minimal and populated template
+   databases to simplify setup and testing.
+3. **End-to-End and Compatibility Testing**: Includes comprehensive testing
+   using the [Smoke Test
+   Utility](/reference/ods-api/platform-dev-guide/utilities/smoke-test-utility)
+   and [Bulk Load
+   Utility](/reference/ods-api/platform-dev-guide/utilities/bulk-load-client-utility).
+
+📖 [View Full
+Changelog](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/compare/v0.5.0...v0.6.0)
+
+## 🚀 Features Delivered So Far
+
+* Resources API
+* Descriptor API
+* Discovery API
+* Dependency API
+* API Extensions
+* PostgreSQL datastore
+* **OpenSearch/Elasticsearch** integration for high-performance search
+* Realtime Streaming via **Kafka**
+* API-driven client credential and claim set management
+* Integration with third-party OAuth Identity Provider (**Keycloak**)
+* Namespace-based authorization
+* Relationship-based authorization (e.g. via student enrollment and staff
+  employment)
+* **Docker images** and **Docker Compose** configurations for easy local and
+  cloud-native deployments
+
+## 🛠️ Try It Out
+
+Download the v0.6.0 release.
+
+```powershell
+git clone https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service
+cd Data-Management-Service
+git checkout v0.6.0
+```
+
+Then, follow the instructions in the [Getting Started with the Ed-Fi Data
+Management Service
+Platform](<https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/GETTING_STARTED.md>)
```

Comment:
What do you think about adding a note to connecting to #ed-fi-dms-feedback in Slack to provide feedback?

(shouldn't that be a public channel rather than private?)